File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (!(height > 0 && height < 9));

    for (int i = 1; i <= height; i++)
    {
        int spaces = height - i;
        for (int a = 1; a <= spaces; a++)
        {
            printf(" ");
        }
        for (int a = 1; a <= i; a++)
        {
            printf("#");
        }
        printf("  ");
        for (int a = 1; a <= i; a++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: week6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height < 9 and height > 0:
        break


for row in range(height):
    for space in range(height - row - 1):
        print(" ", end="")
    for hash in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for hash in range(row + 1):
        print("#", end="")
    print()